src/main/java/uk/gov/companieshouse/accountsDates/AccountsDates.java

```diff
@@ -0,0 +1,199 @@
+package uk.gov.companieshouse.accountsDates;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.Period;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AccountsDates {
+
+	private static final String PERIOD_START = "periodStart";
+	private static final String PERIOD_END = "periodEnd";
+	private static final String DATE_FORMAT_YYYYMMDD = "yyyy-MM-dd";
+	private static final String DATE_FORMAT_D_MMMM_YYYY = "d MMMM yyyy";
+
+	public LocalDate convertStringToDate(String stringDate) {
+		DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_FORMAT_YYYYMMDD);
+		return LocalDate.parse(stringDate, formatter);
+	}
+
+	public String convertDateToString(LocalDate date) {
+		DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_FORMAT_YYYYMMDD);
+		return date.format(formatter);
+	}
+
+	// from 2017-03-05 to 3 March 2005
+	public String convertStringToDisplayDate(String stringDate) {
+
+		DateTimeFormatter dateFormatOriginal = DateTimeFormatter.ofPattern(DATE_FORMAT_YYYYMMDD);
+		DateTimeFormatter dateFormatDesired = DateTimeFormatter.ofPattern(DATE_FORMAT_D_MMMM_YYYY);
+		LocalDate date = LocalDate.parse(stringDate, dateFormatOriginal);
+
+		return date.format(dateFormatDesired);
+	}
+
+	public Map<String, String> getDateAndTime(String dateString) {
```

Comment:
This method is only used on the accounts submission display page where there will always be a date and time passed in 

"closed_at" : ISODate("2018-07-24T09:36:44.262Z"),"

Reply:
Ahh in that case, it will be worth renaming the variable to `dateTimeString` as it has both date and time in it. Also worth adding javadoc to the method saying that it accepts a dateTime string